crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -157,8 +175,35 @@ impl<'w, 's> Commands<'w, 's> {
         }
     }
 
-    /// Equivalent to iterating `bundles_iter` and calling [`Self::spawn`] on each bundle, but
-    /// slightly more performant.
+    /// Spawns entities to the [`World`] according to the given iterator (or a type that can
+    /// be converted to it).
+    ///
+    /// The end result of this command is equivalent to iterating `bundles_iter` and calling
+    /// [`spawn`](Self::spawn) on each bundle, but it is slightly more performant due to memory
```

Comment:
Imo "slightly" is a bit of an understatement. I'd prefer to omit it. This is also faster because it does archetype calculation once, instead of once per-entity.